### PR TITLE
[SHK-280] Member 코드 구조 개선

### PR DESCRIPTION
**Pull request by BnDC**
### ⛏ 작업 사항
- 누락된 bean validation message 작성
- 중복 코드 메서드화
  - getDeliveryEntity
  - getMemberEntity
  - validateAccess 
- 누락된 코드 추가

### 📝 작업 요약
- 코드 구조 개선

### 💡 관련 이슈
- Resolved : [SHK-286], [SHK-287], [SHK-287], [SHK-288], [SHK-290], [SHK-291]

[SHK-286]: https://shoekream.atlassian.net/browse/SHK-286?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[SHK-287]: https://shoekream.atlassian.net/browse/SHK-287?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[SHK-287]: https://shoekream.atlassian.net/browse/SHK-287?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[SHK-288]: https://shoekream.atlassian.net/browse/SHK-288?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[SHK-290]: https://shoekream.atlassian.net/browse/SHK-290?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[SHK-291]: https://shoekream.atlassian.net/browse/SHK-291?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/main/java/com/prgrms/kream/domain/member/controller/MemberController.java [hunk 1]
```diff
@@ -87,50 +87,50 @@ public ApiResponse<String> logoutMember(HttpServletResponse httpServletResponse)
...
+	public ApiResponse<MemberGetResponse> getMember(@PathVariable("memberId") Long id) {
 		return ApiResponse.of(memberFacade.getMember(id));
 	}
 
-	@PostMapping("/{id}")
+	@PostMapping("/{memberId}")
 	@ResponseStatus(OK)
 	public ApiResponse<MemberUpdateResponse> updateMember(
-			@PathVariable Long id,
+			@PathVariable("memberId") Long id,
```
Comment (ttaehee) [inline src/main/java/com/prgrms/kream/domain/member/controller/MemberController.java]:
```suggestion
			@PathVariable Long memberId,
```
요거는 어떨가요??

**Review by ttaehee — APPROVED** [review]
주형님 고생하셨습니다 😀

**Review by goseungwon — APPROVED** [review]
안녕하세요 고승원 입니다. 🌞
코멘트할 부분이 없군요!

File: src/main/java/com/prgrms/kream/domain/member/controller/MemberController.java (hunk 1 above)
Comment (BnDC) [reply src/main/java/com/prgrms/kream/domain/member/controller/MemberController.java]: 수정했습니다~😀

**Pull request merged**